src/function/geometry/distance.js

```diff
@@ -233,6 +223,21 @@ export const createDistance = /* #__PURE__ */ factory(name, dependencies, ({ typ
     return _isNumber(a[0]) && _isNumber(a[1]) && _isNumber(a[2])
   }
 
+  function _isNdim (a) {
```

Comment:
I know that this fits with the naming of `_2d` and `_3d` (and these function names should probably also be changed but this definitely does not need to be done as part of this PR).

I wonder if we could think of a more descriptive name for this function?

Reply:
I think your comment is resolved already? `_2d` and `_3d` are gone, and `_isNdim` sounds pretty descriptive to me.

Reply:
Edit, sorry, you're right these functions are still there. Would make sense to rename them to `is2d` or something like that.